src/app/App.svelte

```diff
@@ -353,7 +352,7 @@
     } else if (document.body.style.position === "fixed") {
       document.body.setAttribute("style", "")
 
-      if (!isNil(scrollY)) {
+      if (scrollY) {
```

Comment:
This is important in the case of scrollY being 0; welshman/lib has an isNil function